packages/icon/src/Icon.spec.tsx

```diff
@@ -1,20 +1,71 @@
 import React from 'react';
+import path from 'path';
+import fs from 'fs';
+import { toJson } from 'xml2json';
 import { render, cleanup } from '@testing-library/react';
 import { typeIs } from '@leafygreen-ui/lib';
 import { createIconComponent } from '.';
 
 afterAll(cleanup);
 
+describe('packages/Icon/glyphs/', () => {
+  const glyphPaths = fs
+    .readdirSync(path.resolve(__dirname, './glyphs'))
+    /**
+     * './glyphs/' contains an index.ts file, so we filter out
+     * anything that's not an svg.
+     */
+    .filter(path => /.*\.svg$/.test(path));
+
+  glyphPaths.forEach(glyphPath => {
+    describe(`${glyphPath}`, () => {
+      const { svg } = require(path.resolve(__dirname, `./glyphs/${glyphPath}`));
+
+      if (!svg) {
+        return;
```

Comment:
Should we fail instead? This shouldn't ever be the case. `expect(svg).not.toBeNull()`?

Reply:
Actually, I think we can get rid of this altogether. We already explicitly filter out non-svg files from `glyphPaths`. I'm just gonna remove this.